TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/includes/ImageHandler.kt

```diff
@@ -25,31 +28,42 @@ import android.view.ViewConfiguration
 import android.widget.ImageView
 import android.widget.RelativeLayout
 import androidx.core.view.drawToBitmap
+import com.microsoft.device.display.samples.twonote.MainActivity
 import com.microsoft.device.display.samples.twonote.NoteDetailFragment
 import com.microsoft.device.display.samples.twonote.structures.SerializedImage
 import java.io.ByteArrayOutputStream
 import java.util.Calendar
 import kotlin.math.abs
 import kotlin.math.max
+import kotlin.math.roundToInt
 import kotlin.math.sqrt
 
 class ImageHandler(private val fragment: NoteDetailFragment) {
     private val names: MutableList<String> = mutableListOf()
     private val images: MutableList<ImageView> = mutableListOf()
     private val compressedImages: MutableList<String?> = mutableListOf()
+    private val rotations: MutableList<Boolean> = mutableListOf()
     private var space = 0f
     private var prevHeight = 0
     private var prevWidth = 0
     private var clickStartTime = 0L
+    private var deleteMode = false
 
     // Create a new ImageView and add it to the container
-    fun addImageToView(uri: Uri) {
+    fun addImageToView(uri: Uri, isRotated: Boolean, x: Float? = null, y: Float? = null, w: Int? = null, h: Int? = null) {
```

Comment:
can we remove the extra parameters here? I don't think we're using them now that image upload isn't a thing